### PR TITLE
Stac 11 event notification

### PR DESCRIPTION
**Pull request by gggina**

### CODE REVIEW

File: use/health-state-and-event-notifications/anomaly-detection-with-baselines.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ This baseline functions works well for stationary metrics, for example data cent
 
 ## Check for anomalies on a baseline metric stream
 
-Once you have [added a baseline](anomaly-detection-with-baselines.md#add-a-baseline-to-a-metric-stream) to a metric stream and you see the baseline bounds drawn on top the metric stream chart you can now configure a check to alert on anomalies.
+Once you have [added a baseline](anomaly-detection-with-baselines.md#add-a-baseline-to-a-metric-stream) to a metric stream and you see the baseline bounds drawn on top the metric stream chart you can now configure a check to send event notifications on anomalies.
```
Comment (craffit) [inline use/health-state-and-event-notifications/anomaly-detection-with-baselines.md:132]: 'change the health state'

Comment (craffit) [inline use/health-state-and-event-notifications/anomaly-detection-with-baselines.md:132]: 'change health states'

Comment (craffit) [reply use/health-state-and-event-notifications/anomaly-detection-with-baselines.md:132]: (and optionally send event notifications)

File: use/introduction-to-stackstate/anomaly-detection.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Read more about the [Autonomous Anomaly Detector \(BETA\)](../../stackpacks/add-
 
 ## Baseline anomaly detection
 
-Each metric stream can have metric baselines manually configured or set by a template. The baselines determine the normal operating values of a metric stream and can be used in health checks and to trigger alerts. Note that any metric stream with a base line configured will not be picked up for autonomous anomaly detection.
+Each metric stream can have metric baselines manually configured or set by a template. The baselines determine the normal operating values of a metric stream and can be used in health checks and to trigger event notifications. Note that any metric stream with a base line configured will not be picked up for autonomous anomaly detection.
```
Comment (craffit) [inline use/introduction-to-stackstate/anomaly-detection.md:24]: health state changes

File: use/introduction-to-stackstate/stackstate_architecture.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ A Script and Query Language provides access to all dimensions of the 4T Data Mod
 
 REST APIs are available for external services and are also used by our Command Line Interface. Every user interface is kept up to date by Websocket APIs.
 
-Alerts, notifications, tickets, webhooks, and API calls are just a few examples of output data sources to let you respond to
+Event notifications, tickets, webhooks, and API calls are just a few examples of output data sources to let you respond to
```
Comment (craffit) [inline use/introduction-to-stackstate/stackstate_architecture.md:17]: just notifications

File: configure/telemetry/checks_and_streams.md [hunk 4]
```diff
@@ -38,20 +38,20 @@ Telemetry streams are supplied via plugins. Different plugins provide one or mul
...
 
-Read how to [add a telemetry stream to an element](../../use/health-state-and-alerts/add-telemetry-to-element.md) or how to [add telemetry during topology synchronization](telemetry_synchronized_topology.md).
+Read how to [add a telemetry stream to an element](../../use/health-state-and-event-notifications/add-telemetry-to-element.md) or how to [add telemetry during topology synchronization](telemetry_synchronized_topology.md).
 
 ### Baselines
 
-A baseline can be attached to a metric stream. The baseline consists of an average, a lowerDeviation and a higherDeviation for batches of metric data. Checks can use the baseline values on a metric stream to trigger an alert if a batch of metrics deviates from the baseline.
+A baseline can be attached to a metric stream. The baseline consists of an average, a lowerDeviation and a higherDeviation for batches of metric data. Checks can use the baseline values on a metric stream to trigger an event notification if a batch of metrics deviates from the baseline.
```
Comment (craffit) [inline configure/telemetry/checks_and_streams.md:45]: health state changes

File: use/health-state-and-event-notifications/anomaly-detection-with-baselines.md (hunk 1 above)
Comment (gggina) [reply use/health-state-and-event-notifications/anomaly-detection-with-baselines.md:132]: updated

File: use/introduction-to-stackstate/anomaly-detection.md (hunk 2 above)
Comment (gggina) [reply use/introduction-to-stackstate/anomaly-detection.md:24]: updated

File: use/introduction-to-stackstate/stackstate_architecture.md (hunk 3 above)
Comment (gggina) [reply use/introduction-to-stackstate/stackstate_architecture.md:17]: updated

File: configure/telemetry/checks_and_streams.md (hunk 4 above)
Comment (gggina) [reply configure/telemetry/checks_and_streams.md:45]: updated

**Pull request merged**